src/Sylius/Bundle/WebBundle/Behat/WebContext.php

```diff
@@ -468,9 +468,9 @@ public function iAmOnTheOrderPage($action, $number)
      */
     public function iShouldBeOnTheOrderPage($action, $number)
     {
-        $this->iAmOnTheOrderPage($action, $number);
+        $product = $this->findOneBy('product', array('name' => $name));
```

Comment:
There's no name in this method, I think it's the change for `iShouldBeOnTheProductPage($name)` at line 447.